backend/Platform/urls.py

```diff
@@ -13,6 +13,7 @@
     path("accounts/", include("accounts.urls", namespace="oauth2_provider")),
     path("options/", include("options.urls", namespace="options")),
     path("identity/", include("identity.urls", namespace="identity")),
+    path("healthbackend/", include("health.urls", namespace="healthbackend")),
```

Comment:
instead of `healthbackend:health`, what if we rename it to `healthcheck:backend`